frontend/src/components/Dialog/Dialog.vue

```diff
@@ -35,29 +35,13 @@
     </vue-final-modal>
 </template>
 
-<script>
-export default {
-    inheritAttrs: false,
-    emits: ['confirm'],
```

Comment:
Hi Nick, saw it's trying to emit the "confirm" method, we might need to declare this emit in order to call this method. https://vuejs.org/guide/typescript/composition-api.html#typing-component-emits